src/__tests__/Login.test.js

```diff
@@ -1,32 +1,60 @@
 import React from 'react';
 import faker from 'faker';
 import { render, fireEvent, waitFor } from '@testing-library/react';
+import { useSelector } from 'react-redux';
 import Login from 'Containers/Login';
-import store from 'Store/store';
-import { Provider } from 'react-redux';
-import { BrowserRouter, Router } from 'react-router-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+jest.mock('react-redux', () => ({
+  useDispatch: jest.fn(),
+  useSelector: jest.fn(),
+  useHistory: () => ({
+    push: jest.fn(),
+  }),
+}));
 
 describe('Register', () => {
```

Comment:
En lugar de hacer el mockup del use selector en todos los test, podemos hacer un beforeEach como en Register y Orders y sacar el mocke de los test indivudales
```suggestion
describe('Register', () => {
beforeEach(() => {
    useSelector.mockImplementation((selector) => selector({
      logUser: {
        data: { documentNumber: '232323' },
        loggedIn: false,
      },
    }));
  });
```